template-files/error.html

```diff
@@ -0,0 +1,131 @@
+    <link rel="stylesheet" href="styles/nav.css" />
+    <link rel="stylesheet" href="styles/base.css" />
+
+    <!-- SCRIPTS -->
+    <script   src="https://code.jquery.com/jquery-2.2.4.min.js"   integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44="   crossorigin="anonymous"></script>
+
+    <script>
+        var url = window.location.href;
+        url = url.toLowerCase();
+        /* Add site url below */
+        if (url != 'https://stn.wim.usgs.gov/error.html') {
+
+            var request = new XMLHttpRequest();
+            request.open('GET', url, true);
+            request.onload = function() {
+              if (request.status >= 200 && request.status < 400) {
+                // Success!
+                window.location = url;
+              } else {
+                // We reached our target, but it returned an error
+                //window.location = '/';
+                window.history.pushState(null, "", '/error.html');
+              }
+            };
+            request.onerror = function() {
+              // There was a connection error of some sort, or does not exist
+              //window.location = '/';
+              window.history.pushState(null, "", '/error.html');
+            };
+            request.send();
+        }
+    </script>
+
+</head>
+<body>
+
+    <!-- USGS top bar, universal for USGS sites -->
+    <div id="usgscolorband">
+        <div id="usgsbanner">
+
+            <div class="max_wrap">
+
+                <div id="usgsidentifier"><a href="https://www.usgs.gov/"><img src="imgs/usgslogo.png" alt="USGS - science for a changing world" title="U.S. Geological Survey Home Page" width="178" height="72" /></a></div>
+                <div id="usgsccsabox">
+                    <div id="usgsccsa" style="text-align:right;">
+                    	<br /><a href="https://www.usgs.gov/">USGS Home</a>
+                    	<br /><a href="https://answers.usgs.gov/">Contact USGS</a>
+                    	<br /><a href="https://search.usgs.gov/">Search USGS</a>
+                		<br />
+                    </div>
+                </div>
+
+            </div>
+
+        </div>
+    </div>
+
+    <nav class="nav-main">
+
+        <a href="/#/">
+            <span class="nav-brand full-only"><img src="imgs/wimvector.png">Web Informatics and Mapping</span>
+            <span class="nav-brand mobile-only"><img src="imgs/wimvector.png">WiM</span>
+        </a>
+        <i class="ion-navicon mobile-only mobile-menu-btn"></i>
+
+
+
+    </nav>
+
+
+
+    <div class="error-box">
+
+        <h1>Error. Page not found.</h1>
+        <h2>Something went wrong. Please check to be sure you have entered your URL correctly.</h2>
+        <h3>You may be looking for <a href="#">put your site here</a>.</h3>
+
+    </div>
+
+
+
+
+    <footer class="footer">
+
+        <p class="address">
+            WIM @ Upper Midwest Water Science Center
+            <br>
+            8505 Research Way, Middleton, WI 53562
```

Comment:
This address should be changed to:
1 Gifford Pinchot Drive, Madison WI 53726

Reply:
Unless @HansVraga  thinks it should be Mounds View/St.Paul since that is where he is. ether way, the Middleton office is no more.